main/com.tunein/manifest.tt

```diff
@@ -0,0 +1,111 @@
+#[subcategory="media"]
+{
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+  
+  entity station #_[description="Radio stations on TuneIn"];
+  entity show #_[description="Show names from the station on TuneIn"];
+
+  list query station(out id: Entity(com.tunein:station)
+                     #[unique=true]
+                     #[string_values="com.tunein:station"]
+                     #_[canonical={
+                       default="base",
+                       base=["name", "title"],
+                       property=["name #", "title #", "channel #"],
+                       passive_verb=["named #", "called #", "titled #"],
+                     }],
+                     out show: Entity(com.tunein:show)
+                     #[unique=true]
+                     #[string_values="com.tunein:show"]
+                     #_[canonical={
+                       default="base",
+                       base=["name", "title", "track", "show", "program", "current show", 
+                             "current program", "now playing", "currently playing"],
+                       passive_verb=["named #", "called #", "titled #", "related to #"],
+                       property=["name #", "title #", "now playing #", "currently playing #"]
+                     }],
+                     out link: Entity(tt:url)
+                     #[filterable=false]
+                     #_[canonical=["link", "url"]],
+                     out image: Entity(tt:picture)
+                     #[filterable=false]
+                     #_[canonical={
+                       default="base",
+                       base=["picture", "image", "photo", "cover"]
+                     }]
+  )
+  #_[canonical=["radio channel", "radio show", "radio station"]]
+  #_[canonical_short=["radio"]]
+  #_[on_error={
+    query_invalid=["radio station does not exist"],
+    bad_request=["malformed radio station name"],
+    unauthorized=["radio service is not accessible without a valid user name and password"],
+    service_unavailable=["radio service is temporarily not available"],
+    network_error=["network_error"]
+  }]
+  #[minimal_projection=["id"]]
+  #_[formatted=[{
+    type="rdl",
+    displayTitle="${id}",
+    webCallback="${link}",
+    displayText="Now playing ${show}",
+    pictureUrl="${image}"
+  }]]
+  #[doc="search a specific internet radio channel from TuneIn"];
+  #[confirm=false];
+
+  action play (in req id: Entity(com.tunein:station)
+               #[string_values="com.tunein:station"]
+               #_[prompt=["which channel would you like to listen to"]]
+               #_[canonical={
+                 default="base",
+                 base=["name", "title"],
+                 property=["name #", "title #", "channel #"],
+                 preposition=["named #", "called #", "titled #"],
+               }]
+  )
+  #_[canonical=["listen to radio", "broadcast radio", "stream radio", "tune radio"]]
```

Comment:
You should choose a verb that goes well with "Ok, so you would like me to ..." as the first canonical (so "stream" or "play"), it will improve the naturalness of agent replies.